smallfile-wrapper/publisher.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import redis
+import time
+import traceback
+import sys
+
+redis_host = sys.argv[1]
+number_of_pods = sys.argv[2]
+
+try:
+    r = redis.StrictRedis(host=redis_host, port=6379)
+    count = 0
+    while int(count) != int(number_of_pods):
+        count = r.execute_command("PUBSUB NUMSUB smallfile")[1]
+        print(count, file=sys.stdout)
```

Comment:
just curious about the need the need to explicitly specify file as `sys.stdout`, because from my understanding the default file object to a print function us `sys.stdout` based on [documentation](https://docs.python.org/3/library/functions.html#print)

Reply:
I didn't know about that, for now , I am just replacing print() with sys,stdout.write() , keeping it irrespective of python version used.

Reply:
print() works fine in python3, just can't use it without parentheses